src/sentry/static/sentry/app/components/events/errors.tsx

```diff
@@ -0,0 +1,243 @@
+    const endpoint = `/organizations/${orgId}/issues/${issueId}/reprocessing/`;
+
+    addLoadingMessage(t('Reprocessing issue\u2026'));
+
+    try {
+      await api.requestPromise(endpoint, {
+        method: 'POST',
+      });
+    } catch (e) {
+      clearIndicators();
+      addErrorMessage(
+        t('Failed to start reprocessing. The event is likely too far in the past.')
+      );
+      return;
+    }
+
+    clearIndicators();
+    browserHistory.push(
+      `/organizations/${orgId}/issues/?query=tags[original_group_id]:${issueId}`
+    );
+  };
+
+  onReprocessStart = () => {
+    openModal(this.renderReprocessModal);
+  };
+
+  renderReprocessModal = ({Body, closeModal, Footer}) => (
+    <React.Fragment>
+      <Body>
+        <p>
+          {t(
+            'You can choose to re-process events to see if your errors have been resolved. Keep the following limitations in mind:'
+          )}
+        </p>
+
+        <ul>
+          <li>
+            {t(
+              'Sentry will duplicate events in your project (for now) and not delete the old versions.'
+            )}
+          </li>
+          <li>
+            {t(
+              'Reprocessing one or multiple events counts against your quota, but bypasses rate limits.'
+            )}
+          </li>
+          <li>
+            {t(
+              'If an event is reprocessed but did not change, we will not create the new version and not bill you for it (for now).'
+            )}
+          </li>
+          <li>
+            {t(
+              'If you have provided missing symbols please wait at least 1 hour before attempting to re-process. This is a limitation we will try to get rid of.'
+            )}
+          </li>
+        </ul>
+      </Body>
+      <Footer>
+        <ButtonBar gap={1}>
+          <Button onClick={this.onReprocessGroup}>
+            {t('Reprocess all events in issue')}
+          </Button>
+          <Button style={{marginLeft: space(1)}} onClick={this.onReprocessEvent}>
+            {t('Reprocess single event')}
+          </Button>
+          <Button style={{marginLeft: space(1)}} onClick={closeModal}>
+            {t('Cancel')}
+          </Button>
+        </ButtonBar>
+      </Footer>
+    </React.Fragment>
+  );
+
+  render() {
+    const {event, project} = this.props;
+    // XXX: uniqueErrors is not performant with large datasets
+    const errors =
+      event.errors.length > MAX_ERRORS ? event.errors : this.uniqueErrors(event.errors);
+    const numErrors = errors.length;
+    const isOpen = this.state.isOpen;
+    return (
+      <StyledBanner priority="danger">
+        <BannerSummary>
+          <StyledIconWarning />
+          <span>
+            {tn(
+              'There was %s error encountered while processing this event',
+              'There were %s errors encountered while processing this event',
+              numErrors
+            )}
+          </span>
+          <a data-test-id="event-error-toggle" onClick={this.toggle}>
```

Comment:
I would use the button component here with the priority link.

Reply:
This code is from the old jsx file, I won't change the UI here